assets/css/base/_base.scss

```diff
@@ -846,9 +846,7 @@ select,
 textarea {
 	font-size: 100%; /* Corrects font size not being inherited in all browsers */
 	margin: 0; /* Addresses margins set differently in IE6/7, F3/4, S5, Chrome */
-	vertical-align: baseline; /* Improves appearance and consistency in all browsers */
-	/* stylelint-disable */
-	*vertical-align: middle; /* Improves appearance and consistency in all browsers */
+	vertical-align: middle; /* Improves appearance and consistency in all browsers */
```

Comment:
According to https://developer.mozilla.org/en-US/docs/Web/CSS/vertical-align, `vertical-align: baseline;`  is the default value and when using `vertical-align: middle;` instead, I cannot see a change either. Wouldn't it make more sense to completely remove this definition from the current statement? 

@haszari  or @senadir Is one of you aware why the definition above had been added in the first place?